scripts/setup-credentials.sh

```diff
@@ -74,6 +74,10 @@ github_pr_creator_org: "10gen"
 expansions:
   papertrail_key_id: $PAPERTRAIL_KEY_ID
   papertrail_secret_key: $PAPERTRAIL_SECRET_KEY
+  aws_key: $AWS_ACCESS_KEY_ID
+  aws_secret: $AWS_SECRET_ACCESS_KEY
```

Comment:
Are these meant to be different than the ones used on line 54 and 55? I don't have time for a full review so I may be missing something.

Reply:
I'm not sure where those are coming from honestly -- I'm using the credentials we get from assume role at the beginning of the task!